iothub/device/src/Transport/Mqtt/MqttTransportSettings.cs

```diff
@@ -26,6 +26,10 @@ public class MqttTransportSettings : ITransportSettings
         private const QualityOfService DefaultReceivingQoS = QualityOfService.AtLeastOnce;
         private static readonly TimeSpan DefaultConnectArrivalTimeout = TimeSpan.FromSeconds(10);
         private static readonly TimeSpan DefaultDeviceReceiveAckTimeout = TimeSpan.FromSeconds(300);
+        /// <summary>
+        /// To enable certificate revocation check. Default to be false.
+        /// </summary>
+        public bool CertificateRevocationCheck = false;
```

Comment:
Flipped default value to false based on following facts: 
- avoid break change to customer
- based on this [article](https://medium.com/@alexeysamoshkin/how-ssl-certificate-revocation-is-broken-in-practice-af3b63b9cb3)
> There is no ready-to-go 100% working and reliable solution to make browsers detect revoked certificates in a timely manner and refuse connecting